tests/test_unit/test_filtering.py

```diff
@@ -74,7 +74,7 @@ def test_filter_by_confidence(sample_dataset, caplog):
             ).values[:, 0]
         )
     )
-    assert n_nans == 3213
+    assert n_nans == 2555
```

Comment:
Because I actually changed the underlying dataset! Before we had "DLC_single-mouse_EPM" and "SLEAP_single-mouse_EPM", but they did not actually correspond to the same video. I updated the DLC one so it matches the SLEAP one (enabling us to use the same video and frame.